lib/connect_four.rb

```diff
@@ -1,2 +1,14 @@
+require './lib/game'
+
 p "Welcome to CONNECT FOUR"
 p "Enter p to play. Enter q to quit."
+
+def user_response
+  p "Enter p to play. Enter q to quit."
+    user_input = gets.chomp
```

Comment:
We need to account for "invalid"(not p or q) inputs.
It may help to do:
```ruby
user_input = nil
until user_input.downcase == "g" || user_input.downcase == "q"
user_input = gets.chomp
end

```